sssom/parsers.py

```diff
@@ -618,6 +618,14 @@ def to_mapping_set_document(msdf: MappingSetDataFrame) -> MappingSetDocument:
                 ok = False
                 if k:
                     k = str(k)
+                if (
+                    (hasattr(Mapping, k) or hasattr(MappingSet, k))
+                    and v is not None
+                    and "|" in str(v)
```

Comment:
is coercing into a string so smart? what type is `v` supposed to be? maybe it would be better to add a check for whether it's multivalued into that class (and also the function for parsing)

Reply:
I dont know what your comment exactly refers to, but if it would be better, we could add another intsanceof str check to be sure.. What class should the multivalued check be in?

Reply:
Idk - it's not obvious what a `v` is by reading this code. Obviously a better variable name is required as a first step. Second, if it's ever possible a `v` isn't a string already, I would be incredibly worried to just do a string coercion, since every object can do this and it will get totally meaningless results in most cases. Better to make a helper function that is a "is_multivalued()` that can do instance checks

Reply:
Ok, I will work this out with @hrshdhgd later today.